src/dotnet/Chat.UI.Blazor/Components/ChatLanguageToggle.razor

```diff
@@ -6,10 +6,13 @@
 @{
     var language = State.LatestNonErrorValue.LanguageOrDefault();
 }
-<button class="chat-language-toggle relative h-8 w-8 p-1 rounded-full text-md font-semibold text-secondary hover:text-toggle-on"
-        @onclick="Toggle">
+
+<Button Class="chat-language-toggle relative h-8 w-8 p-1 rounded-full text-md font-semibold text-secondary hover:text-toggle-on"
```

Comment:
1. Move `h-8 w-8 rounded-full text-md font-semibold text-secondary` to `chat-language-toggle` CSS - i.e. leave just positioning classes here.

2. Let's agree extra `on` class = something is on "on" state, so `hover:text-toggle-on` should be actually in CSS: `.hover:chat-language-toggle.on`